burger/src/components/Main/index.js

```diff
@@ -9,7 +9,7 @@ import {
   initialIngridients,
 } from "./Ingridients/reducerIngridients";
 
-const Main = () => {
+const Main = ({ setModal, setModalIngr, setBurgerPrice }) => {
   const [ingidients, setIngidients] = useReducer(
```

Comment:
`useReducer`? 🤔 

There is not a reason to use it unless you have redux. Which you do not